modules/sample/src/test/ts/client/AsyncPassTest.ts

```diff
@@ -1,10 +1,24 @@
 import { UnitTest } from '@ephox/bedrock-client'
 
-UnitTest.asynctest('AsyncPass Test', (success, failure) => {
-
+UnitTest.asyncTest('AsyncPass Test 1', (success, failure) => {
   new Promise(function (resolve, reject) {
```

Comment:
Interesting that this didn't have a poyfill previously... it really should have.

Reply:
Yeah, looks like CI is only running these tests in phantomjs.

Reply:
I've logged internal ticket TINY-6159 to add tests cross-browser.

Reply:
When I add `@ephox/wrap-promise-polyfill` and try to import it, I get:

```
TS2529: Duplicate identifier 'Promise'. Compiler reserves name 'Promise' in top level scope of a module containing async functions.
```

Reply:
I'm going to add the polyfill fix to TINY-6159. Sound good?

Reply:
That's a typescript error? weird, we don't have that issue in other tests, oh well.

Reply:
As discussed: polyfill has been added. It doesn't work with `async..await` - seems like TS requires a global `Promise` for that. For now, I've removed all `async..await`.